Dockerfile

```diff
@@ -6,7 +6,9 @@ RUN apt-get update && \
     apt-get clean
 
 # Install Python requirements
+RUN pip install web.py
```

Comment:
Hmm isn't this a dependency of Bottle. It should be covered by there. Generally speaking, we should have all `pip` requirements inside `requirements.txt`.